recipes/solweig-gpu/meta.yaml

```
@@ -0,0 +1,63 @@
+package:
+  name: solweig-gpu
+  version: "1.0.1"
+
+source:
+  url: https://files.pythonhosted.org/packages/de/83/0d408ff5f63b0f42441899e202b444c12a4326179ed352f4fc6534a11969/solweig_gpu-1.0.1.tar.gz
+  sha256: 437b78bae73ba49ac1c7dfeeb8f2176e7f674fabca4450c4c1fc6ccdd93f3b31
+
+build:
+  script: python -m pip install . --no-deps -vv
+  number: 0
+  ignore_run_exports:
+    - _openmp_mutex
+
+requirements:
+  host:
+    - python
+    - pip
+    - setuptools
+    - gdal
+
+  run:
+    - python
+    - pip
+    - numpy
+    - numba
+    - pytorch
+    - scipy
+    - pandas
+    - netcdf4
+    - pytz
+    - shapely
+    - timezonefinder
+    - gdal
+    - xarray
+    - tqdm
+    - pyqt
+
+test:
+  imports:
+    - solweig_gpu
+  commands:
+    - python -m pip check
+    - python -c "import subprocess; subprocess.call(['thermal_comfort', '--help'])"
```

```suggestion
    - pip check
    - thermal_comfort --help
  requires:
    - python {{ python_min }}
    - pip
```

Slight style nit for using `pip check` instead of `python -m pip check